mempool/src/tests/mod.rs

```diff
@@ -16,7 +16,6 @@ mod node;
 mod shared_mempool_test;
 
 pub mod fuzzing;
-#[cfg(any(feature = "fuzzing", test))]
 pub mod mocks;
```

Comment:
I can't see why this needed to be removed: the whole mod `tests` is feature-gated the same way in the library. This should not affect anything, but it's neater to just check out the whole file from upstream.

Reply:
That's been added back in, not sure why it was there. Thanks for checking this.